Resources/Prototypes/Reagents/narcotics.yml

```diff
@@ -314,6 +315,26 @@
   physicalDesc: reagent-physical-desc-acrid
   flavor: bitter
   color: "#215263"
+  metabolisms:
+    Narcotic:
+      effects:
+      - !type:MovespeedModifier
+        walkSpeedModifier: 0.65
+        sprintSpeedModifier: 0.65
+      - !type:HealthChange
+        damage:
+          types:
+            Poison: 2
+      - !type:GenericStatusEffect
+        key: SeeingRainbows
+        component: SeeingRainbows
+        type: Add
+        time: 10
+        refresh: false
+      - !type:ChemVomit # Vomiting is a symptom of brain damage
+        probability: 0.05
+      - !type:Drunk # Headaches and slurring are major symptoms of brain damage, this is close enough
+        boozePower: 20
```

Comment:
Please lower this. It felt like this was way longer than 20 seconds.

Reply:
Yeah, I kind of had the feeling it was a bit too strong. Do note that booze power isn't proportional to drunkenness time. I've just quartered the booze power, so now impedrezene is only around 5 times as strong as most default alcohols, such as beer.